src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -252,11 +252,11 @@ class ClientChannelControlHelper
 // Synchronous callback from chand->resolving_lb_policy to process a resolver
 // result update.
 static bool process_resolver_result_locked(
-    void* arg, const grpc_channel_args& args, const char** lb_policy_name,
+    void* arg, grpc_core::Resolver::Result* result, const char** lb_policy_name,
     grpc_core::RefCountedPtr<LoadBalancingPolicy::Config>* lb_policy_config) {
   channel_data* chand = static_cast<channel_data*>(arg);
   chand->have_service_config = true;
-  ProcessedResolverResult resolver_result(args, chand->enable_retries);
+  ProcessedResolverResult resolver_result(result, chand->enable_retries);
```

Comment:
Similar issue here

Reply:
and looks like in many of the client_channel files

Reply:
and tons of conflicts exist now

Reply:
Fixed issue by rebasing.